shared/templates/template_OVAL_accounts_password

```diff
@@ -0,0 +1,61 @@
+<def-group>
+  <definition class="compliance" id="accounts_password_pam_%VARIABLE%" version="3">
+    <metadata>
+      <title>Set Password %VARIABLE% Requirements</title>
+      <affected family="unix">
+        <platform>multi_platform_rhel</platform>
+        <platform>multi_platform_fedora</platform>
+      </affected>
+      <description>The password %VARIABLE% should meet minimum requirements</description>
+    </metadata>
+{{% if product == "rhel6" %}}
+    <criteria>
+        <criterion comment="rhel6 pam_cracklib %VARIABLE%" test_ref="test_password_pam_cracklib_%VARIABLE%" />
+    </criteria>
+{{% else %}}
+    <criteria operator="AND" comment="conditions for %VARIABLE% are satisfied">
+      <extend_definition comment="pwquality.so exists in system-auth" definition_ref="accounts_password_pam_pwquality" />
+      <criterion comment="pwquality.conf" test_ref="test_password_pam_pwquality_%VARIABLE%" />
+    </criteria>
+{{% endif %}}
+  </definition>
+
+{{# Only credit related variables allow negative values #}}
+{{% if "credit" in "%VARIABLE%" %}}
+  {{% set negative_symbol = "-?" %}}
```

Comment:
negative_symbol is a very misleading name for this variable. I suggest "sign" as in signed and unsigned numbers.